### ISSUE TITLE
Error with file name containing foreign characters

### ISSUE THREAD

**Issue by gregoriopellegrino** [vaites/php-apache-tika#28]
I realized that the command comes out with error code 1 when I ask to extract the text of a file called `07.-아름다운GIRL-Who-are-You-Official-Lyrics.pdf`

I did some debugging and the problem seems to be related to the lack of language setting in the environment.

I modified the file `src/CLIClient.php` in lines 316 and later, with the code:
```
$env = array(
     'LANG' => 'en_US.UTF-8'
);
$process = proc_open($command, $descriptors, $pipes, $env);
```

Now it works properly.

**Comment by vaites** [comment]
Thanks @gregoriopellegrino, I can simply add a new parameter to set this value. Will make some tests too...

Anyway, can you please tell me what's the operating system and version you're using?.

**Comment by gregoriopellegrino** [comment]
CentOS 7

**Comment by vaites** [comment]
OK, it's enough for my tests and will keep you informed. One last question: what's your actual $LANG value?. In theory, the library must use all env variables of the user that runs the command. How are you running it?

**Comment by gregoriopellegrino** [comment]
In the CLI I have `en_US.UTF-8`, so I set the same in the script since I've seen that in CLI the command worked with no errors.

**Comment by vaites** [comment]
My assumptions seems to be wrong.

**Comment by vaites** [comment]
I've just uploaded to the 1.x branch the changes to allow users to set its own environment variables:
```php
$client = Client::make('/path/to/tika-app.jar');
$client->setEnvVars(['VARIABLE' => 'value']);
```
Can you please check if it works for you (set `dev-1.x` as version on your `composer.json`). It still needs some testing but i think it must work without issues.

**Comment by vaites** [comment]
Will release soon the 1.1 version with this feature...

**Comment by vaites** [comment]
Fixed with v1.0.2 release
